src/components/sharing/ShareBox/__tests__/ShareBox-test.js

```diff
@@ -1,23 +1,22 @@
-'use strict';
+jest.disableAutomock();
 
-jest.autoMockOff();
+import React from 'react';
+import ShareBox from '../';
+import ShareIcon from '../../ShareIcon/';
+import TestUtils from 'react-addons-test-utils';
 
 describe('Share Box', function() {
-  var React                  = require('react');
-  var ShareBox               = require('../');
-  var ShareIcon              = require('../../ShareIcon/');
-  var TestUtils              = require('react-addons-test-utils');
-  var scryRenderedComponents = TestUtils.scryRenderedComponentsWithType;
+  let scryRenderedComponents = TestUtils.scryRenderedComponentsWithType;
 
-  var mockServices = [
+  let mockServices = [
```

Comment:
I agree, but also, it's like 500 lines to change with only marginal benefit in the existing code...

(i.e. 9/10 it's a thing that is defined then immediately used, once, which a const *would* guard extra errors on but mostly won't ever be accidentally re-used.)